docs/src/quickstart.md

```diff
@@ -0,0 +1,93 @@
+# Quickstart
+
+## Writing your first kernel
+
+Kernel functions are marked with the [`@kernel`](@ref). Inside the `@kernel` macro
+you can use the [kernel language](@ref api_kernel_language). As an example, the `mul2`` kernel
+below will multiply each element of the array `A` by `2`. It uses the [`@index`](@ref) macro
+to obtain the global linear index of the current work item.
+
+```julia
+@kernel function mul2_kernel(A)
+  I = @index(Global)
+  A[I] = 2 * A[I]
+end
+```
+
+## Launching kernel on the host
+
+You can construct a kernel for a specific backend by calling the kernel
+function, with the first argument being the device of type `KA.Device`, the
+second argument being the size
+of the workgroup and the third argument being a static `ndrange`. The second and
```

Comment:
I don't know if it's mentioned elsewhere, but a quick dictionary might me nice:

1. A **workgroup* is also called a *block* in NVIDIA CUDA and designates the number of threads acting in parallel. For the GPU, this is usually something like 256, for the CPU, this is usually less than or equal to your number of cores
2. An **ndrange** is also called a *grid* in NVIDIA CUDA and designates the total number of items to work on. If you are using a workgroup of size 1 (non-parallel execution), the ndrange is the number of items to iterate over in a loop.

Reply:
Great, I took your snippet here and rephrased it a bit. For now, it's in the quickstart. Maybe at some point it should become a sperarate page.